src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
That makes sense. A breaking change doesn't necessarily have to involve an API that we document and explicitly support. Thanks @ljharb 👍 

So yeah, I think implementing `node()` and `nodes()` would be a good idea for the next major release.